src/transformers/models/maskformer/convert_maskformer_original_pytorch_checkpoint_to_pytorch.py

```diff
@@ -0,0 +1,728 @@
+# coding=utf-8
+# Copyright 2022 Facebook AI Research and The HuggingFace Inc. team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# import sys
+# import sys
+from argparse import ArgumentParser
+from dataclasses import dataclass
+from pathlib import Path
+from pprint import pformat
+from typing import Any, Dict, Iterator, List, Set, Tuple
+
+import torch
+import torchvision.transforms as T
+from PIL import Image
+from torch import Tensor
+
+# The MaskFormer dir is present in the following directory, I need to add it to the path so python can see it
+# sys.path.append("/home/zuppif/Documents/Work/hugging_face/")
```

Comment:
There needs to be specific directions for a user, for instance:
```suggestion
# The MaskFormer should be present in the following directory, or adapt this path to your needs.
sys.path.append("../")
```
which implies it's at the same level as the Transformers repo (anything without your name in it works as a default ;-) )

Alternatively, we could have an `arg` for this path and the sys path in the main function, along with the imports there (defining `add_mask_former_config` and `OriginalMaskFormer` as None at the start of the file to have them as global variables).

Reply:
adding `sys.path.append("../")` breaks `make quality`, what do you think is the best strategy here (if we don't want to add the `sys path` as a paremeter